packages/platforms/browser/tests/auto-instrumentation/full-page-load-plugin.test.ts

```diff
@@ -39,6 +40,10 @@ describe('FullPageLoadPlugin', () => {
     performance.addEntry(createPerformancePaintTimingFake({ startTime: 128 }))
     performance.addEntry(createPerformanceEventTimingFake({ startTime: 0.4, processingStart: 1 }))
 
+    let appState: AppState = 'starting'
+    const setAppState = jest.fn((state: AppState) => {
+      appState = state
+    })
```

Comment:
are you not able to get these from the `plugins: (spanFactory)` of the test client on line 57? they should be part of the base client so no need to mock them here